src/App.tsx

```diff
@@ -1,16 +1,28 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [updatedMovies, setUpdatedMovies] = useState(moviesFromServer);
+
+  const addMovie = (newMovie: Movie) => {
+    setUpdatedMovies(oldMovies => [...oldMovies, newMovie]);
+  };
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={updatedMovies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie
+          onAdd={newMovie => {
+            addMovie(newMovie);
+          }}
```

Comment:
Maybe I'm mistaken but you can write just 
```suggestion
          onAdd={addMovie}
```